Add section ASGI for `@now/python` (#942)
zeit/docs: pages/docs/v2/deployments/official-builders/python-now-python.mdx

The example above can be seen live as <https://simple-python-flask.now-examples.now.sh/Hello>

## Asynchronous Server Gateway Interface

The Asynchronous Server Gateway Interface (ASGI) is a calling convention for web servers to forward requests to asynchronous web applications written in Python. You can use ASGI with frameworks such as [Sanic](https://sanic.readthedocs.io).

Instead of defining a `handler`, define an `app` variable in your Python file.

For example, define a `index.py` file inside a folder as follows:

```python
from sanic import Sanic
from sanic.response import json
app = Sanic()

@app.route('/')
@app.route('/<path:path>')
async def index(request, path=""):
    return json({'hello': path})
```

Inside `requirements.txt` define:

```
sanic==19.6.0
```

> And define a `now.json` like:

Reviewer: ```suggestion
And define a `now.json` like so:
```

```json
{
  "version": 2,
  "builds": [{ "src": "index.py", "use": "@now/python" }],
  "routes": [{ "src": "(.*)", "dest": "index.py" }]
}
```

> Most frameworks use their own implementation of routing. However, you can use a catch-all route to circumvent the framework and instead use [Now Routing Layer](/docs/v2/deployments/routes) to match a route to a Lambda.

Reviewer: ```suggestion
Most frameworks use their own implementation of routing. However, you can use a catch-all route to circumvent the framework and instead use the [Now Routing Layer](/docs/v2/deployments/routes) to match a route to a serverless function.
```

> The example above can be seen live as <https://simple-python-sanic-rf2yqvtg9.now.sh/world>

Reviewer: ```suggestion
The example above can be seen live at <https://simple-python-sanic-rf2yqvtg9.now.sh/world>
```
